src/components/editTextField/editTextField.scss

```
@@ -0,0 +1,46 @@
+$background-color:#fff;
+$border-color:#757575;
+.user-info {
+    width: 100%;
+    height: 40px;
+    background-color: $background-color;
+    display: inline-flex;
+    border: none;
+    border-bottom: 0.1px solid $border-color;
+}
+
+.user-info__name {
+    width: 50%;
+    background-color: $background-color;
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+}
```

Можно использовать вложенность, которую предоставляет scss
```suggestion
.user-info {
    width: 100%;
    height: 40px;
    background-color: $background-color;
    display: inline-flex;
    border: none;
    border-bottom: 0.1px solid $border-color;
    
    &__name {
    width: 50%;
    background-color: $background-color;
    display: flex;
    justify-content: flex-start;
    align-items: center;
}
}

```